online/apply_functions/bbci_apply_sendControl.m

```diff
@@ -16,72 +16,80 @@
 
 %% - INIT
 if isequal(control_signal, 'init'),
-  switch(bbci_feedback.receiver),
-   case '',
-    % do nothing
-   case 'udp',
-    if ~exist('pnet', 'file'),
-      global BTB
-      addpath(fullfile(BTB.PrivateDir, 'import/tcp_udp_ip'))
+    switch(bbci_feedback.receiver),
+        case '',
+            % do nothing
+        case 'udp',
+            if ~exist('pnet', 'file'),
+                global BTB
+                addpath(fullfile(BTB.PrivateDir, 'import/tcp_udp_ip'))
+            end
+            pnet('closeall');
+            data_feedback.opt.socke= pnet('udpsocket', 1111);
+            pnet(data_feedback.opt.socke, 'udpconnect', 'localhost', 9100');
+        case 'pyff',
+            %    send_udp_xml('init', bbci_feedback.host, bbci_feedback.port);
+            pyff_sendUdp('init',  bbci_feedback.host, bbci_feedback.port);
+        case 'tobi_c',
+            send_tobi_c_udp('init', bbci_feedback.host, bbci_feedback.port);
+        case 'lsl',
+            lib = lsl_loadlib();
```

Comment:
Since LSL is not part of the BBCI toolbox, we need to do something here. The preferred way is to include the LSL toolbox in the
  bbci_import_dependencies.m
if that is not done yet.

Insert
  util_loadLSL();
above and add a function like the following to the toolbox:
function util_loadLSL()
if ~exist('lsl_loadlib', 'file'),
  global BTB
  lslDir= fullfile(BTB.Dir, 'external/liblsl-Matlab');
  if exist(lslDir, 'dir'),
    addpath(genpath(lslDir));
  else
    error(sprintf('LSL library expected in %s.\nDownload manually or use bbci_import_dependencies(''LSL')', lslDir));
  end
end

The same function should be called in bbci_acquire_lsl.m